02 Toolbar/src/components/toolbar/buttons/linkToolbarButton.tsx

```diff
@@ -0,0 +1,40 @@
+import * as React from 'react';
+import {textAreaTool} from '../../../common/ui/tools/textAreaTool';
+import {BaseToolbarButton} from './baseToolbarButton';
+
+export class LinkToolbarButton extends BaseToolbarButton {
```

Comment:
Maybe, It is a better name like `SelectingTextToolbarButton` or `SelectingTextButton`